internal/gatewayapi/translator_test.go

```diff
@@ -76,6 +77,19 @@ func TestTranslate(t *testing.T) {
 
 			got := translator.Translate(resources)
 
+			envoyGatewayNsName := "envoy-gateway-gateway-1"
```

Comment:
this shouldnt be needed, we address the order issue here https://github.com/envoyproxy/gateway/blob/main/internal/gatewayapi/sort.go